app/views/front_ui/_maps.html.erb

```diff
@@ -9,27 +9,27 @@
   <% maps.each do |map| %>
       <% warp = map.placed_warpables.first %>
     <div class="card-deck map col-md-6 col-lg-3 col-xs-12 mb-2 ml-2">
-        <div class="card bg-transparent card-without-border">
+        <div class="card bg-transparent card-without-border ">
         <a href="/maps/<%= map.slug %>">
             <% if warp.present? %>
             <img class="card-img-top map-img" src="<%= warp.image.url  %>" alt="<%= map.name %>">
```

Comment:
This is the image displayed when a map has an image, kindly check if the two will have the same feel as the default one.

Reply:
@cesswairimu I had made the changes in the conditional statement. Kindly review it ! Thank You!